app/models/account.rb

```diff
@@ -322,6 +323,14 @@ def save_with_optional_media!
     save!
   end
 
+  def hides_followers?
+    hide_collections || user_hides_network?
```

Comment:
For boolean attributes, the getter with `?` is generated automatically and is the preferred access method. It should be `hide_collections?`